flowermd/utils/rigid_body.py

```diff
@@ -0,0 +1,134 @@
+import gsd
+import gsd.hoomd
+import hoomd
+import numpy as np
+from cmeutils.geometry import moit
+
+
+def _get_com_mass_pos_moi(snapshot, rigid_const_idx):
```

Comment:
Small preference change, but can we move the hidden methods below the non hidden ones?